### PR TITLE
Add rules for Youtube to safe search list

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/issues/167

### CODE REVIEW

**Review by Alex-302 — APPROVED** [review]

File: assets/engines_safe_search.txt [hunk 1]
```diff
@@ -257,3 +257,10 @@
 |yandex.tj^$dnsrewrite=NOERROR;A;213.180.193.56
 |yandex.tm^$dnsrewrite=NOERROR;A;213.180.193.56
 |yandex.uz^$dnsrewrite=NOERROR;A;213.180.193.56
+#
+# Youtube
+|www.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|m.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtubei.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtube.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|www.youtube-nocookie.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
```
Comment (Alex-302) [inline assets/engines_safe_search.txt:266]: Wrong file.
Correct:
https://adguardteam.github.io/HostlistsRegistry/assets/youtube_safe_search.txt

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: assets/youtube_safe_search.txt [hunk 2]
```diff
@@ -5,3 +5,8 @@
 |www.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
 |youtube.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
 |youtubei.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|www.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|m.youtube.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtubei.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|youtube.googleapis.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
+|www.youtube-nocookie.com^$dnsrewrite=NOERROR;CNAME;restrictmoderate.youtube.com
```
Comment (Alex-302) [inline assets/youtube_safe_search.txt:12]: Literally the same rules

Comment (Sergey-Lyapin) [reply assets/youtube_safe_search.txt:12]: Lol)
 Maybe @Mizzick want us to duplicate them in engines_safe_search.txt?

Comment (Mizzick) [reply assets/youtube_safe_search.txt:12]: Indeed I wanted these rules in `engines_safe_search.txt` as I haven't seen that you have them already in the `youtube_safe_search.txt` file.

Then it's fine, no need to change anything.

**Pull request closed without merging**